src/picai_prep/mha2nnunet.py

```diff
@@ -126,11 +150,14 @@ def process_and_write(self, scans_out_dir: Path, annotations_out_dir: Path):
             atomic_image_write(sample.lbl, path=annotations_out_dir / f"{self.subject_id}.nii.gz", mkdir=True)
             self.write_log(f'Wrote annotation to {destination_path}')
 
-    def compile_log(self):
+    def compile_log(self) -> Optional[str]:
         if self.settings.verbose == 0:
             # logging is disabled
             return None
 
+        if self.skip_conversion:
+            return f"Skipping {self.subject_id}, already converted."

```

Comment:
maybe only for settings.verbose == 1?

Reply:
I meant to write 2

Reply:
I think it's useful to log that a case is skipped, also to inform the user that we do not overwrite the images.

Reply:
I agree, but verbosity level 1 should only include failures. We even wrote it in the documentation stub:
```
- verbose: control logfile verbosity. 0 does not output a logfile,
                    1 logs cases which have critically failed, 2 logs all cases (may lead to
                    very large log files)
                    Default: 1
```